### ISSUE TITLE
[BUG] Using host_group Module to create multiple Groups for Customer Global doenst work

### ISSUE THREAD

**Issue by checker788** [Checkmk/ansible-collection-checkmk.general#685]
**Describe the bug**
When Using the host_group Module to create multiple Groups at once the customer cannot be set to "global"

**Component Name**

Component Name: host_group

**Ansible Version**

```console
$ ansible --version
ansible [core 2.16.11]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/noah/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /var/ansible/repository/ansible_collections
  executable location = /usr/bin/ansible
  python version = 3.12.3 (main, Sep 11 2024, 14:17:37) [GCC 13.2.0] (/usr/bin/python3)
  jinja version = 3.1.2
  libyaml = True
```

**Checkmk Version and Edition**

```console
2.3.0p10 CME
```

**Collection Version**

```console
$ ansible-galaxy collection list

```

**To Reproduce**
Create Task with Host_group Module

**Expected behavior**

Create Host Groups in for Global Customer

**Actual behavior**

Creates Host Groups for Provider Customer

**Minimum reproduction example**

```yaml
---
- name: Create Host Groups
   hosts: all
   tasks:
   - name: "Create several host groups."
      checkmk.general.host_group:
       server_url: "http://myserver/"
           site: "mysite"
           automation_user: "myuser"
           automation_secret: "mysecret"
           customer: "global"
           groups:
           - name: "my_host_group_one"
              title: "My Host Group One"
           - name: "my_host_group_two"
              title: "My Host Group Two"
        state: "present"
```

**Comment by msekania** [comment]
@checker788, yes indeed there is a bug in module for the multiple host_groups scenario.

I'll make PR with bug fixes

**Comment by robin-checkmk** [comment]
I did not even have the time to reproduce the issue and @msekania here already build the fix. Impressive reaction time, buddy! :muscle: I'll see to it, that the PR is reviewed and merged, and a bug fix release will be made. Thanks for reporting @checker788! :pray:

**Comment by robin-checkmk** [comment]
Fix released with `5.3.1`.
